src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -613,16 +660,11 @@ private async Task DebugVerifyNoErrorsAsync(MutableConflictResolution conflictRe
                             }
                             else
                             {
-                                var newMetadataName = symbol.ToDisplayString(s_metadataSymbolDisplayFormat);
-                                var oldMetadataName = conflictAnnotation.RenameDeclarationLocationReferences[symbolIndex].Name;
-                                if (newLocation == null ||
-                                    newLocation.IsInSource ||
-                                    !HeuristicMetadataNameEquivalenceCheck(
```

Comment:
This is the only conflict check I don't think can expand to the multiple symbol rename scenario.